client/my-sites/pages/style.scss

```diff
@@ -95,12 +98,12 @@
 	text-align: left;
 }
 
-.page__info {
+.pages__blog-posts-page__info {
 	color: $gray;
 	font-size: 12px;
 	max-width: 300px;
-	float: right;
```

Comment:
This PR introduces the first user-visible change, and there will be more on the way which we want to surface by flipping a switch (feature flag). I am fine with the info label position change not being behind a flag since it's not a functional change.